jupyter/car-iar-poisson/car-iar-poisson.ipynb

```diff
@@ -0,0 +1,1310 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting the Model to the Scotland Lip Cancer Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cmdstanpy\n",
+    "from cmdstanpy import CmdStanModel\n",
+    "import numpy as np\n",
```

Comment:
it would be good to list the Python packages this notebook will be using.

Reply:
I have added the Python packages in the markdown block.